plugins/openldap/check-syncrepl.rb

```diff
@@ -87,17 +87,22 @@ class CheckSyncrepl < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i)
 
   def get_csns(host)
+    port = config[:port]
+    if host.include? ":"
+       host, port = host.split(":")
```

Comment:
None of them should be empty as,
* config[:port] has a default value
* host comes from iterating over config[:hosts], so the function `get_csns` is only called when this list has elements
* The split is only applied if the split character is presentl, so it is ensured that host and port are populated.

Reply:
I think that we could have "some-host:" (with semmicolon, but without port), and this would split into "some-host" and "". But I don't really think we need to be so deep in these validations. Nevermind ;)